enterprise/coderd/workspaces_test.go

```diff
@@ -191,6 +191,51 @@ func TestCreateWorkspace(t *testing.T) {
 		require.Equal(t, http.StatusBadRequest, apiErr.StatusCode())
 		require.Contains(t, apiErr.Message, "doesn't exist")
 	})
+
+	// Auditors cannot "use" templates, they can only read them.
+	t.Run("Auditor", func(t *testing.T) {
```

Comment:
This passes without an error on `main`. Now auditors get a 403 Forbidden if they do not have `use` perms from somewhere else